htdocs/expedition/class/expedition.class.php

```diff
@@ -1105,6 +1105,8 @@ function fetch_lines()
 				$obj = $this->db->fetch_object($resql);
 
                 $line->line_id          = $obj->line_id;
+                $line->rowid            = $obj->line_id;    // TODO deprecated
```

Comment:
In dolibarr objects have an id field, no rowid, nore line_id.
I suggest using also a expedition class -> id